tests/v2/test_waku_rln_relay.nim

```diff
@@ -259,6 +285,37 @@ procSuite "Waku rln relay":
671e26", "5c9af59018e86e3b717a88b2c08e1b6e9a44b5fe64adc25ac1751c8182eeb312", "4196b9532b071f04bdf27837026beb8b062a7c7de33f9e7bcb5b05d8651bdf07", "2700d1680cdfd84a4941610608bff1bdc330ae8a94be24567c189864b266c616", "3f90fa45595427e189067fc5d1b8bb21cbfc4d54530aa3b78266587d2239f41c", "8c80075af391945307627bbc4a4ea6406bd57608445748ece82ef0bae037f60e", "b0c798a4bd153e9420be8211f4360d79d7c3479316b98b548c416c491e80dc1a", "cf2a0dadea865534d471efb19d4001801e5e5695342c7e198a46abfd51bc2b25", "edc85e19f95f66e8d7f2b49e6d00d0473dac9efedee53cbbee578391ce7b9f05", "cebad61c29ce8a4482287b737a7688325571d045b814a966a7ba5a307503dc0f", "214dda0412bef8e0b1d53b2b056091a9c7355c8dbf2f11048a2f3737fb005828", "219961a17263ae29db21cc32589271ccf6dc3416740daafde9a4d93f0416e817", "d6d68ec08be07fa3f5e85497ba6f5a042a18c5ea543da8f9f6615839d0748f05", "ceef793da479045c207be9f827d10dbba1d1637cf0a2d4100d34bad53d82d32f", "62f191cfd6b0888fa0fd3c31597884b7ccdee6c74a255a4afc0f0f25a4ba532d", "b2e58d8678bb66fc00d02047b985cbe5efa1f0e22c71af9684d67fcb1cad940d", "c3ee9ede913d1987b28eeec5de8cc8c0778d1f6e43c4f7ac28c2e5c681571702", "e7e484487cafa71e9c1231740f64bc662b6794ef6ea376c1989074a56cb42c1d", "912e8b04d865dd91f4548849523c91444ac70ff589ba298ce10678a5816ace2d", "2ba9baa6ff79c7727ad1648c1c7fcc4e9943d8b1a2c68f6c1ef1e70d70877510", "e59707ec253abeb04e6cceaf803dd1b69030f7ac8b916059b0d7d60edacbae26", "f36327129c4a66cf7f976b11c0705bf642740bd90055df4ec1540e6b550f5b03", "448025c0ed6a60edd0fa97eb1f84a1f04c8416eb15268b2a89c24a47dee7132b", "08e26120d9d2937b05d58c08aff93266317f19df353258290da4fb068e0af02f", "bb8668525fef05da280502c1353e0c257dce8cfd53beadde87e4366a5a79dc24", "e6f256845d0b8f9177727723857e9b30314cfc753432ca365c0e6b100d68fd22", "642a784417b01858ffed0ccd9b4967a9e7980facdce7667a07b367fabe63d225", "5ebd45b2dc7e010782a4c51a8ae18bad00c35597010a2107cad5041ffab69329", "d73e8c0e36e5588836bc9be4820e108d008f8399e4a9bacb5a452d4850a72d10", "01b81a331bc0c7818812c3848d382551183b35a16d74df5c8c32d561edb74b1c", "403e3cb57be08aef37dcc8af0a93a6ea8064540c8bcdb76fb34875c3d894be03", "6e420bc668f7b2aeee7e719daa3a7a4471bd04e8bcf24a27e4a5ae07e8c44207", "98baf438da4c655b9d92dc9033ae0f5cd7f0efda9b0d1bb894d08f1ce8620a01", "1282dda3446fb157e236155197def0d309eecb9dde8445f599ead69d4f31d916", "6453a3cf73ab5f3bafe645ce69fcdb612526fa66659e564c783c5600fb3a400f", "a8c1bfdbae2f56d10ccaf89887286c7c3256f7bdba21e98ac3c44578926c642c", "34bd74a58df94ae160f90bb64d202908d95b9dd0d89f02cd7e6bc8ddaf11280f", "a7377899f99a93796cd8901e17357403a900570d869fc49afed5e0e8f4bb281e", "63485fb688def0de08ae284fbc0cb6b7d9a6696e9542eb9328d69de33607f902", "6767b23b636e9b89c8b3ac3614fbdf529b6485cb3fc8ae110b705dbcb9e1fe12", "b54c6a95a349be5a1c09e487b4149111c6ccfbdaec5ec6f8ff82fb21ee894113", "604b619329ec489482dcc4af6f5a02f9ed1418f624f900eca143355127406217", "a556c534da5f5549bceb1ba95535dd57af5fd59011772552d5fea78a02e78a11", "34eae6950d3227d848e7e075aecdb65791306156b6503a89406e5802bfd0e117", "a6f3a081dd622f89f0e8703281052ff2a275507214fed44d3813a5027e5c762a"]
```

Comment:
Wouldn't it be possible to generate this `seq` in a test utility function? I imagine it should even be possible to write a repeatable generator with the same result each time.
Otherwise, perhaps the `seq` can be extracted as a `const` in a separate module and reused between tests, because I see it's initialized elsewhere as well?

Reply:
The underlying key generation is non-deterministic and each time produces a different result, I also do not have access to the randomness used in that proc to be able to control the output. But I can certainly use the `const` idea, Thanks! done in this [commit](https://github.com/status-im/nim-waku/pull/718/commits/24126e875185422d16e502f22528d319ae541cb4)